Dalamud/Utility/Util.cs

```diff
@@ -634,13 +634,24 @@ public static unsafe void WriteAllBytesSafe(string path, byte[] bytes)
 
         if (!Windows.Win32.PInvoke.FlushFileBuffers(tempFile))
             throw new Win32Exception();
-        
+
         tempFile.Close();
 
         if (!Windows.Win32.PInvoke.MoveFileEx(tempPath, path, MOVE_FILE_FLAGS.MOVEFILE_REPLACE_EXISTING | MOVE_FILE_FLAGS.MOVEFILE_WRITE_THROUGH))
             throw new Win32Exception();
     }
 
+    /// <summary>
+    /// Turns a RGBA color value into ABGR for usage in ImGui.PushColor
+    /// </summary>
+    /// <param name="rgba">Color value with byte order of RGBA.</param>
+    /// <returns>The color value in byte order of ABGR, the format that ImGui uses.</returns>
+    public static uint RGBAToABGR(uint rgba)
+    {
+        var tmp = ((rgba << 8) & 0xFF00FF00) | ((rgba >> 8) & 0xFF00FF);
+        return (tmp << 16) | (tmp >> 16);
+    }
```

Comment:
That's just an endian swap. .NET already provides [BinaryPrimitives.ReverseEndianness](https://learn.microsoft.com/en-us/dotnet/api/system.buffers.binary.binaryprimitives.reverseendianness). I remember @KazWolfe linking it in Discord.

Reply:
Yeah it is just an endian swap, if wanted i can use the .net method, tho i think having a Util explicitly named by what it is used for in dalamud can provide some benefits

Reply:
Yeah, I get it. Although it can also be used as ABGRToRGBA. 🤷‍♂️ It's up to Team Dalamud, I guess. 😄

Reply:
Besides, I think this would fit better in [Dalamud/Interface/ColorHelpers.cs](https://github.com/goatcorp/Dalamud/blob/master/Dalamud/Interface/ColorHelpers.cs). 🙂

Reply:
I think a utility function is fine, but please do use the endian swap rather than re-implement our own. I'd probably want a better name though, seeing as this is technically bidirectional.

Reply:
Which name would you suggest instead?